### PR TITLE
ODATA-1411

### PR DESCRIPTION
**Pull request by ralfhandl**
https://issues.oasis-open.org/browse/ODATA-1411

### CODE REVIEW

File: odata-protocol/11 Data Service Requests.md [hunk 1]
```diff
@@ -386,11 +386,10 @@ The value of the `$expand` query option is a comma-separated list of
 navigation property names, stream property names, or `$value` indicating
 the stream content of a media-entity.
 
-For navigation properties, the navigation property name is optionally
```
Comment (GeraldKrause) [inline odata-protocol/11 Data Service Requests.md:389]: The text in this paragraph does not match with the new paragraph you added after this one. I think you wanted to remove the paragraph above this one instead. Kindly check.

**Review by GeraldKrause — CHANGES_REQUESTED** [review]

File: odata-protocol/11 Data Service Requests.md (hunk 1 above)
Comment (ralfhandl) [reply odata-protocol/11 Data Service Requests.md:389]: Thanks, I also wanted to remove the paragraph above, done now.

**Review by GeraldKrause — CHANGES_REQUESTED** [review]

File: odata-protocol/11 Data Service Requests.md [hunk 2]
```diff
@@ -382,15 +382,9 @@ The `$expand` system query option indicates the related entities and
...
-the stream content of a media-entity.
-
-For navigation properties, the navigation property name is optionally
-followed by a `/$ref` path segment or a `/$count` path segment, and
-optionally a parenthesized set of [expand options](#ExpandOptions) (for
-filtering, sorting, selecting, paging, or expanding the related
-entities).
+The value of `$expand` is a comma-separated list of expand items. Each
+expand item is evaluated relative to the retrieved resource being
+expanded.
```
[the remark this answers is not in the document]
Comment (ralfhandl) [reply odata-protocol/11 Data Service Requests.md:387]: I intentionally removed the redundant text and think the reference to URL Conventions is better than copying text that will get outdated again.

[the remark this answers is not in the document]
Comment (GeraldKrause) [reply odata-protocol/11 Data Service Requests.md:387]: I fully agree with that! Then I suggest to clarify where to find further details - see my suggestion below.

File: odata-protocol/11 Data Service Requests.md [hunk 3]
```diff
@@ -382,15 +382,9 @@ The `$expand` system query option indicates the related entities and
...
 
 For a full description of the syntax used when building requests, see
 [OData-URL](#ODataURL).
```
Comment (GeraldKrause) [inline odata-protocol/11 Data Service Requests.md:390]: Clarify where to find further details:
```suggestion
[OData-URL](#ODataURL), section 5.1.3.
```

**Review by GeraldKrause — CHANGES_REQUESTED** [review]

**Review by GeraldKrause — APPROVED** [review]

**Review by mikepizzo — APPROVED** [review]

**Pull request merged**